src/viam/components/camera/camera.py

```diff
@@ -48,6 +50,21 @@ async def get_image(self, mime_type: str = "", *, timeout: Optional[float] = Non
         """
         ...
 
+    @abc.abstractmethod
+    async def get_images(self, *, timeout: Optional[float] = None, **kwargs) -> Tuple[List[NamedImage], ResponseMetadata]:
+        """Get simultaneous images from different sensors, along with associated metadata.
+        This should not be used for getting a time series of images from the same sensor.
+
+        Returns:
+            Tuple[List[NamedImage], datetime]:
+                - List[NamedImage]:
+                  The list of images returned from the camera system.
+
+                - ResponseMetadata:
```

Comment:
@bhaney is the idea that timestamp should always be passed? the issue with returning response metadata is that it is optional in responsemetadata. this also generally differs from the response in the rdk/cpp sdk

Reply:
timestamp is definitely optional - there may be situations where that info is not available

Reply:
cool